lib/foods.js

```diff
@@ -1,65 +1,75 @@
 $('document').ready(function() {
+  for (var i = 0; i < localStorage.length; i++){
```

Comment:
@case-eee just want to double check to make sure I completely understand localStorage. In a way it functions like a session in a Rails App so I can call it anywhere and we will have access to what is in it correct?